config/config.go

```diff
@@ -746,6 +750,39 @@ func (re *Regexp) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	return nil
 }
 
+type Matchers []labels.Matchers
```

Comment:
I think now that we have figured out the unmarshaling below, this can just be `type Matchers labels.Matchers` (we unmarshal the whole list of strings in L758 now).

Reply:
makes sense